szacks-building-cohesive-remote-teams.md

```diff
@@ -0,0 +1,202 @@
+A team composed of people with similar backgrounds will have a much narrower range of perspectives than a team made up of people with different backgrounds. 
+This diversity enables the team to look at a situation through a multi-faceted lens, giving them a wider degree of understanding of the big picture.
+
+Diversity is one of the few components of cohesive teams that is potentially *stronger* in a remote environment. 
+That is, distributed teams can have members anywhere; teams comprised of people from multiple backgrounds, countries, and cultures have the ability to work together and increase the value they provide. 
+A team whose members learn to interact with each other and feed off of each other's strengths will be able to blend into a cohesive mesh.
+
+## Regular feedback
+Feedback is one critical aspect of a team's ability to constantly improve. 
+Team members must understand what they're doing right so they can *continue* doing that.
+They must also understand the mistakes they've made so that they can work on those issues. 
+Feedback can come from multiple source and travel in muultiple directions. 
+Managers who oversee a problem with people working together can suggest methods of improvement. 
+Teammates can make suggestions to each other regarding problems they've seen and suggestions on how to improve. 
+Customers can provide feedback on the improvements that will improve their lives.
+
+Managing these feedback loops is much more difficult for remote, distributed teams. 
+Fewer interactions between team members means certain people might not feel comfortable giving constructive criticism.
+(See the section above, on informal interactions!)
+Leaders and managers don't always observe team member interactions and may not be aware that some of the team members are not working well together. 
+The feedback loop has to become an object of deliberte structure and care (more than, say an informal discussion in the cafeteria).
+
+To accomplish this, team leaders and managers should hold regular one-on-one meetings with team members.
+Everyone should attend these meetings with a list of both task- and career-based objectives.
+During these meetings, leaders should provide feedback about the tasks for which the team member is responsible and ask team members should feedback on those tasks, on the team leader, and the team's general direction. 
+This should be an open conversation, where all parties feel safe and valued. 
+
+## Effective retrospectives
+When a project reaches a milestone—or wraps up—it's generally a good idea to conduct a retrospective (or "post-mortem") to review how the team succeeded on the project and how it may need to improve in the future. 
+This involves analyzing the _entire_ project process—both the "good" aspects as well as "bad." 
+An effective retrospective process enables team members to provide feedback to their peers and for everyone to learn from one another.
```

Comment:
During this retrospective, each team member should learn about the methodologies used by each other. It will often point out strengths and weaknesses as well as gaps and collaboration opportunities. By learning all of this, and building bridges to mitigate problems for the future, this will greatly increase the team's cohesiveness.

Reply:
Spot on. That's exactly what the section needed. Done and done.